src/pages/en/guides/cms/storyblok.md

```diff
@@ -22,17 +22,14 @@ To get started, you will need to have the following:
 
 2. **A Storyblok account and space** - If you don't have an account yet, [sign up for free](https://app.storyblok.com/#/signup) and create a new space.
 
-3. **Storyblok API tokens** - You can find and generate your API tokens in the Access Tokens tab of your Storyblok space settings. 
-    - **Preview token** - This token will be used in development to fetch drafts or unpublished versions of your content.
-    - **Public token** - This token will be used in production builds to fetch only published content.
+3. **Storyblok Preview token** - This token will be used to fetch drafts and published versions of your content. You can find and generate your API token in the Access Tokens tab of your Storyblok space settings.
```

Comment:
If we always use the preview token, can users somehow filter out their drafts in production when they fetch that data?

Reply:
Yes, the preview token allows filtering for published and draft content by adding the query params `version=draft/published` which can be used in conjunction with `import.meta.env.DEV` outside of the config file 🎉 ([docs](https://www.storyblok.com/docs/api/content-delivery#topics/introduction)).

Reply:
Sorry I missed this @kevinzunigacuellar. That’s great!

Do you think we should add a little mention of that somewhere in that case?

> To hide draft posts in production add the `version=published` query param to this request: …example…

Reply:
@delucis Yes, I have included this `query` param in the code snippets for both the integration setup and the tutorial. I mentioned what it does in the tutorial but not in the integration set up. I think it would be okay to repeat this.

Reply:
D’oh! Didn’t actually check you hadn’t already done this, sorry @kevinzunigacuellar!

In that case, I think this PR LGTM!

Reply:
To be quite honest, I have no idea why I included it. I seem like a good idea at the time. Happy it came in handy.